build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'com.spidasoftware.releaseNotes'
 apply plugin: 'org.owasp.dependencycheck'
 
 group = 'com.spidasoftware'
-version = '12.0.0-SNAPSHOT' // If you're changing the major version also change the currentVersion in ConverterUtils.
+version = '12.0.1-SNAPSHOT' // If you're changing the major version also change the currentVersion in ConverterUtils.
```

Comment:
use 11.0.3 for publishing the artifact to Artifactory
use 12.0.0-SNAPSHOT for merge to master